package.json

```diff
@@ -39,7 +39,7 @@
     "karma-coverage-istanbul-reporter": "~2.0.1",
     "karma-jasmine": "~1.1.2",
     "karma-jasmine-html-reporter": "^0.2.2",
-    "ng-packagr": "^4.2.0",
+    "ng-packagr": "^10.1.1",
```

Comment:
`@angular-stdlib` only supports **Angular v7** right now. `ng-packagr@10.1.1` is only compatible with **Angular** ≥ **v10.**
The latest version of `ng-packagr` compatible with **Angular v7** is [5.7.1](https://www.npmjs.com/package/ng-packagr/v/5.7.1.).